tests/integration/models/reddit/test_modmail.py

```diff
@@ -34,6 +34,28 @@ def test_mute(self, _):
             conversation = self.reddit.subreddit('all').modmail('ik72')
             assert conversation.user.mute_status['isMuted']
 
+    @mock.patch('time.sleep', return_value=None)
+    def test_read(self, _):
+        self.reddit.read_only = False
+        conversation = self.reddit.subreddit('all').modmail('ik72')
+        with self.recorder.use_cassette('TestModmailConversation.test_read'):
+            conversation.read()
+            conversation = self.reddit.subreddit('all').modmail('ik72')
+            assert conversation.last_unread is None
+
+    @mock.patch('time.sleep', return_value=None)
+    def test_read__other_conversations(self, _):
+        self.reddit.read_only = False
+        conversation = self.reddit.subreddit('all').modmail('p8rp')
+        other_conversation = self.reddit.subreddit('all').modmail('p8rr')
+        with self.recorder.use_cassette(
+                'TestModmailConversation.test_read__other_conversations'):
+            conversation.read(other_conversations=[other_conversation])
+            conversation = self.reddit.subreddit('all').modmail('p8rp')
+            other_conversation = self.reddit.subreddit('all').modmail('p8rr')
```

Comment:
Sorry, I was unclear: by "they don't fetch prior to the assertions", I meant that the fetch isn't triggered *until* the lines with the assertions. So they do fetch, it's just that they don't need to be *re*-fetched from the first time the objects are constructed.

However, I see what you mean about keeping the cassette simpler, so I'd just as soon take the extra checks out.